### PR TITLE
[Draft] Add Dredd test for dynamic shape Multiply

### PR DESCRIPTION
**Pull request by qsunki**
This adds Dredd test for dynamic shape Multiply.

ONE-DCO-1.0-Signed-off-by: sunki <ajsthfldu@outlook.com>

### CODE REVIEW

**Comment by qsunki** [comment]
- related : #13998

File: res/TensorFlowLiteRecipes/Inf_Mul_000/test.recipe [hunk 1]
```diff
@@ -0,0 +1,30 @@
+operand {
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Inf_Mul_000/test.recipe:1]: plz split this to another PR

Comment (qsunki) [reply res/TensorFlowLiteRecipes/Inf_Mul_000/test.recipe:1]: Ah, then I'll mark this PR as a draft and create separate PRs.
